### DISCUSSION TITLE
How can I customize 'X-Frame-Options' ?

### DISCUSSION THREAD

**Discussion by bomb-kst** [goauthentik/authentik#6651] [Q&A]
When I use my odic-client I found error `Refused to display 'http://xxx.xxx.xxx.xx/' in a frame because it set 'X-Frame-Options' to 'deny'. ` So I want to fix it.

I'm using docker-compose file from https://goauthentik.io/docker-compose.yml, please guide me to solve this.

**Comment by BeryJu** [comment]
You can use https://goauthentik.io/docs/installation/configuration#custom-python-settings and set `X_FRAME_OPTIONS = "SAMEORIGIN"` in that file
